operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -206,6 +235,235 @@ local filename = 'mimir-writes.json';
         )
       )
     )
+    .addRowIf(
+      $._config.show_ingest_storage_panels,
+      ($.row('Ingester (ingest storage: fetching and processing records)'))
+      .addPanel(
+        $.timeseriesPanel('Responses / sec') +
+        $.panelDescription(
+          'Responses / sec',
+          |||
+            Rate of responses from Kafka brokers. Client can return multiple responses ("fetches") at once. Some of the responses may be failures.
+          |||
+        ) +
+        $.queryPanel(
+          [
+            |||
+              sum (rate (cortex_ingest_storage_reader_fetches_total{%s}[$__rate_interval]))
+              -
+              sum (rate (cortex_ingest_storage_reader_fetch_errors_total{%s}[$__rate_interval]))
+            ||| % [$.jobMatcher($._config.job_names.ingester), $.jobMatcher($._config.job_names.ingester)],
+            'sum (rate (cortex_ingest_storage_reader_fetch_errors_total{%s}[$__rate_interval]))' % [$.jobMatcher($._config.job_names.ingester)],
+            // cortex_ingest_storage_reader_read_errors_total metric is reported by Kafka client.
+            'sum (rate (cortex_ingest_storage_reader_read_errors_total{%s}[$__rate_interval]))' % [$.jobMatcher($._config.job_names.ingester)],
+          ],
+          [
+            'fetches',
+            'failed',
+            'read errors',
+          ],
+        ) + $.aliasColors({ failed: '#FF0000', 'read errors': '#FF0000' }) + $.stack,
+      )
+      .addPanel(
+        $.timeseriesPanel('Records per fetch') +
```

Comment:
Instead of this panel I would add "Kafka records / sec", which similar to "Kafka fetches / sec" but based on `cortex_ingest_storage_reader_records_*` metrics. I think it's important to show the rate of records / s, as much as the rate of fetches / s.